docs/tutorials/scenario-simulation/installation.md

```diff
@@ -0,0 +1,58 @@
+# Installation
+
+This document contains step-by-step instruction on how to build [AWF Autoware Core/Universe](https://github.com/autowarefoundation/autoware) with `scenario_simulator_v2`.
+
+## Prerequisites
+
+1. Ubuntu 20.04 machine
+2. Hardware with CUDA 11.1 capable graphics card
+3. ROS2 Galactic Geochelone desktop version [installed](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html) and sourced:
+
+## How to build
```

Comment:
To reduce maintenance cost,
How about referring to the [installation page](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/) and writing only the diff on this page?

Reply:
I was about to make this exact same comment. Since Autoware needs to be built anyway, the only remaining tasks would be to import the simulator.repos file and then run colcon build to build the simulator specific components right?

Reply:
We need to run `rosdep install` again.

Reply:
Ah okay. That still reduces the installations steps down to three which is pretty short!